dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/layers/Conv2D.scala

```diff
@@ -9,8 +9,8 @@ class Conv2D private[api] (override val underlying: PyConv2D) extends Layer(unde
   def filters: Int = underlying.filters
   def kernelSize: Int | (Int, Int) = underlying.kernel_size
   def strides: Int | (Int, Int) = underlying.strides
-  def padding: String = underlying.padding
-  def dataFormat: Option[String] = underlying.data_format
+  def padding: Padding = Padding.valueOf(underlying.padding)
+  def dataFormat: Option[DataFormat] = pyOption2Option(underlying.data_format).map(DataFormat.valueOf)
```

Comment:
Maybe sth like `underlying.data_format.toScalaOption` instead?

Reply:
(implicit class or whatever the name in Scala 3... extension method?)

Reply:
`toScalaOption` works fine. I dont know why `toPythonOption` can't figure out the type ;-;